models/api.models.js

```diff
@@ -9,13 +9,27 @@ exports.selectTopics = () => {
   });
 };
 
-exports.selectArticleById = (articleId) => {
-  return db
-    .query("SELECT * FROM articles WHERE article_id = $1", [articleId])
-    .then(({ rows }) => {
+exports.selectArticleById = (article_id) => {
```

Comment:
This model should not support two endpoints. There should be a function for an article by id and an a model for all articles. I would also at this point have a model file for the articles separate from the other models. Same for the controller.